relayer/spy_relayer/src/__tests__/integration.ts

```diff
@@ -73,12 +74,16 @@ describe('Alephium to Ethereum', () => {
       const attestResult = await attestFromAlph(
         alphWallet,
         ALPH_TOKEN_BRIDGE_ID,
-        ALPH_TEST_TOKEN_ID,
+        ALPH_TOKEN_ID,
+        18,
+        'ALPH',
+        'ALPH',
```

Comment:
[Nit] Use info from token list?

Reply:
Yes, in the latest code, we import the ALPH token info from the token-list.